docs/book/src/cronjob-tutorial/testdata/project/.github/workflows/test-e2e.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         run: kind version
 
       - name: Create kind cluster
-        run: kind create cluster
+        run: kind create cluster --name project-test-e2e
```

Comment:
Hey! Thanks for the PR 🙌

I don't think we should make this change directly here.

If we want to create a cluster name, I think it would make more sense to handle that inside something like `setup-test-e2e`. Then `test-e2e` can call `setup-test-e2e` as needed.

So the change here would either:
- Remove the `Create kind cluster` step, **or**
- Replace it with a call to `setup-test-e2e`

Reply:
Yes, agreed - removing it seems like the better option. Other users might rely on different tools, like GitLab, so keeping cluster setup inside `setup-test-e2e` (which is already a dependency of `test-e2e`) makes things more flexible and smoother for everyone.